src/client/utils/tracking/adTraction.ts

```diff
@@ -0,0 +1,93 @@
+import md5 from 'md5'
+import { TypeOfContract } from 'data/graphql'
+import { OfferData } from 'pages/OfferNew/types'
+import {
+  isBundle,
+  isNorwegian,
+  isYouth,
+  isDanishAccidentBundle,
+  isStudentOffer,
+  isDanishTravelBundle,
+} from 'pages/OfferNew/utils'
+import { NoComboTypes, DkBundleTypes } from './tracking'
+
+type TypeOfContractExcludedUnused = Exclude<
+  TypeOfContract | NoComboTypes | DkBundleTypes,
+  'DK_ACCIDENT' | 'DK_ACCIDENT_STUDENT' | 'DK_TRAVEL' | 'DK_TRAVEL_STUDENT'
+>
+
+const adtractionProductMap: Record<TypeOfContractExcludedUnused, number> = {
+  SE_HOUSE: 1477448913,
+  SE_APARTMENT_BRF: 1417356498,
+  SE_APARTMENT_STUDENT_BRF: 1423041022,
+  SE_APARTMENT_RENT: 1417356528,
+  SE_APARTMENT_STUDENT_RENT: 1412601108,
+  NO_HOME_CONTENT_OWN: 1492623645,
+  NO_HOME_CONTENT_RENT: 1492623645,
+  NO_HOME_CONTENT_YOUTH_OWN: 1492623719,
+  NO_HOME_CONTENT_YOUTH_RENT: 1492623719,
+  NO_TRAVEL: 1492623742,
+  NO_TRAVEL_YOUTH: 1492623785,
+  NO_COMBO: 1492623841,
+  NO_COMBO_YOUTH: 1492623841,
+  DK_HOME_CONTENT_OWN: 1589961514,
+  DK_HOME_CONTENT_RENT: 1589961514,
+  DK_HOME_CONTENT_STUDENT_OWN: 1589962112,
+  DK_HOME_CONTENT_STUDENT_RENT: 1589962112,
+  DK_ACCIDENT_BUNDLE: 1589962152,
+  DK_ACCIDENT_BUNDLE_STUDENT: 1613410547,
+  DK_TRAVEL_BUNDLE: 1589962256,
+  DK_TRAVEL_BUNDLE_STUDENT: 1613410728,
+}
+
+export const getBundleAdtractionProductValue = (offerData: OfferData) => {
+  if (isBundle(offerData)) {
+    if (isNorwegian(offerData)) {
+      return isYouth(offerData)
+        ? adtractionProductMap[NoComboTypes.NoComboYouth]
+        : adtractionProductMap[NoComboTypes.NoCombo]
+    }
+
+    if (isDanishAccidentBundle(offerData)) {
+      return isStudentOffer(offerData)
+        ? adtractionProductMap[DkBundleTypes.DkAccidentBundleStudent]
+        : adtractionProductMap[DkBundleTypes.DkAccidentBundle]
+    }
+
+    if (isDanishTravelBundle(offerData)) {
+      return isStudentOffer(offerData)
+        ? adtractionProductMap[DkBundleTypes.DkTravelBundleStudent]
+        : adtractionProductMap[DkBundleTypes.DkTravelBundle]
+    }
+  }
+  return adtractionProductMap[
+    offerData.quotes[0].contractType as TypeOfContractExcludedUnused
+  ]
+}
+
+export const adtraction = (
+  orderValue: number,
+  orderId: string,
+  emailAddress: string,
+  couponCode: string | null,
+  offerData: OfferData,
+) => {
```

Comment:
Agree with what @styrelius usually says that so many positional arguments is less clear than accepting a single object like:

```suggestion
export const adtraction = ({
  orderValue,
  orderId,
  emailAddress,
  couponCode,
  offerData,
}: {
  orderValue: number,
  orderId: string,
  emailAddress: string,
  couponCode: string | null,
  offerData: OfferData,
}) => {
```